app/controllers/application_controller.rb

```diff
@@ -1,3 +1,12 @@
+require "application_responder"
+
 class ApplicationController < ActionController::Base
+  self.responder = ApplicationResponder
+  respond_to :html
+
   protect_from_forgery with: :exception
+
+  def show
+    render :show
+  end
```

Comment:
Instead of having your home page on application controller, a suggestion to create a `HomeController` or `WelcomeController` with a `index` action.

ApplicationController usually gets littered with too many methods. So trying to keep it lean. 
And it is more logical to have one of the others controllers for the home page. It communicates what they do better.

Reply:
Thanks Elle, I like that idea, I will implement a welcome controller I think!